easybuild/easyconfigs/o/OpenFold/OpenFold-1.0.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,91 @@
+easyblock = 'PythonBundle'
+
+name = 'OpenFold'
+version = '1.0.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/aqlaboratory/openfold'
+description = "A faithful PyTorch reproduction of DeepMind's AlphaFold 2"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    ('CMake', '3.20.1'),  # required to build ninja
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('CUDA', '11.3.1', '', True),
+    ('SciPy-bundle', '2021.05'),
+    ('PyYAML', '5.4.1'),
+    ('Biopython', '1.79'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.3.2'),
+    ('Kalign', '3.3.1'),
+    ('UCX-CUDA', '1.10.0', versionsuffix),
+    ('cuDNN', '8.2.1.32', versionsuffix, True),
+    ('NCCL', '2.10.3', versionsuffix),
+    ('dm-tree', '0.1.6'),
+    ('einops', '0.4.1'),
+    ('colossalai', '0.1.8', versionsuffix),
+    ('scikit-build', '0.11.1'),
+    ('OpenMM', '7.5.1', '_AlphaFold'),  # patched OpenMM
```

Comment:
We should update the AlphaFold easyconfigs too to use this?

Maybe it makes sense to flesh that out in a separate PR (which requires this PR), to make it stand out since it's a change against existing easyconfigs...

Reply:
> Also, is it specified somewhere that OpenFold also requires the changes being made in the AlphaFold patch for OpenMM? Maybe add a reference to where that is stated as a comment above this line?

Yes, see e.g. https://github.com/aqlaboratory/openfold/blob/14a79a1205a3e6a065f7ac17f49440f5e64b121f/scripts/install_third_party_dependencies.sh#L23, will add a comment to the easyconfigs